### PR TITLE
feat: Jukebox menu scrolling

### PR DESCRIPTION
**Pull request by ryanzhoudev**

### CODE REVIEW

**Comment by ShadowCat117** [comment]
I think it could be worth adding search support for it too.

Unless the vanilla one has searching, been a while since I used it

**Comment by ShadowCat117** [comment]
There is also a jukebox on housing and it has basically the same layout minus the randomize option however the title is "Select Songs" so maybe support that too

**Review by magicus — APPROVED** [review]

File: common/src/main/java/com/wynntils/models/containers/ContainerModel.java [hunk 1]
```diff
@@ -50,12 +50,14 @@ public final class ContainerModel extends Model {
     private static final Pair<Integer, Integer> CONTENT_BOOK_PREVIOUS_NEXT_SLOTS = new Pair<>(65, 69);
     private static final Pair<Integer, Integer> GUILD_BANK_PREVIOUS_NEXT_SLOTS = new Pair<>(9, 27);
     private static final Pair<Integer, Integer> GUILD_MEMBER_LIST_PREVIOUS_NEXT_SLOTS = new Pair<>(10, 28);
+    private static final Pair<Integer, Integer> JUKEBOX_PREVIOUS_NEXT_SLOTS = new Pair<>(17, 8);
```
Comment (magicus) [inline common/src/main/java/com/wynntils/models/containers/ContainerModel.java:53]: ... wait a sec. Don't you need to reference this as well? You need to add the jukebox screen as scrollable, somehow?

Comment (magicus) [reply common/src/main/java/com/wynntils/models/containers/ContainerModel.java:53]: @DonkeyBlaster 

I did not see that you had auto merge turned on, so this comment arrived too late.

Comment (ryanzhoudev) [reply common/src/main/java/com/wynntils/models/containers/ContainerModel.java:53]: Hmm.... I swear I did the normal implementation and everything and even played with it in-game. Guess not. I'll follow up on this PR with the search and fixes in a bit.

**Pull request merged**